README.md

```diff
@@ -74,7 +74,7 @@ TODO: tested platforms.
 
 ```text
 # Configure build
-$ cmake -S . -B build -DCMAKE_CXX_STANDARD=20
+$ cmake -S . -B build -DCMAKE_CXX_STANDARD=23
```

Comment:
What was the motivation on changing the standard level to 23?  I didn't think this implementation *required* anything in 23 -- except maybe constexpr which probably depends on 26.

Reply:
@JeffGarland `consteval` is the thing

Reply:
ok - should that be conditional in the implementation?

Reply:
No, it will not compile in pre C++23 mode so, C++23 unconditionally.

Reply:
Well, I'm sure with enough macro magic that wouldn't be the case.  Obviously with *degraded functionality*.  And to be clear, not trying to be difficult but just explore what's reasonable/possible.  Anyway, all good please proceed.

Reply:
We changed implementation. C++20 support in doc was for the previous implementation, I must have missed this when switching implementation. I will explore this after #55 is done.